docs/AA--Javascript-in-EE/value-objects/duration.md

```diff
@@ -0,0 +1,473 @@
+
+## Features
+
+- Immutable: manipulating the period of time represented by a `Duration` value object will always return a new instance instead of mutating the existing.
+- Locale aware: For `i18n`, the locale can be provided (eg. `en-ca`) and formatted duration values will use that locale.  Be aware, however, that currently `Duration` value objects only have the locales that are bundled with moment.  Also, for all methods receiving a `locale` argument, the locale defaults to the site's locale if the user is not logged in.  If the user is logged in to the WordPress site, then the default locale will be the user's set locale (not the site's locale).  If the site locale is not valid (because the server has a set locale that moment doesn't have an equivalent for), then the fallback will be `en` and no error will get thrown.
+
+## Installation/Loading:
+
+The `Duration` value object is available via the `eejs` global on its `valueObjects` property. So you can access it directly via `eejs.valueObjects.Duration` provided the bundled `@eventespresso/vo` javascript has been loaded before your script.  For instructions on different ways to do this you can [go here](README.md#Usage).
+
+## API
+
+### CONSTRUCTOR
+
+A `Duration` is constructed with the following two arguments:
+
+- `values`: A simple object of units and their quantities for the period of time the Duration is to reference.  Only the units with values need to be passed.
+- `locale`: This optional argument provides the locale for the Duration.  By default, the locale will be whatever is the current locale for the site (or `en`).  This should be a `BCP47` language tag.  Not all locales are supported.  If the provided language tag is not available, then the value object will attempt to set it to the closest variant.  If that's not available then an `InvalidLocale` error is thrown (an error gets thrown only if client code explicitly provides a locale, relying on the default will not throw an error because if the server locale is not available, it falls back to `en`).
+
+**Example:**
+```js
+// represents `1 year, 2 months of duration`
+const duration = new Duration( { years: 1, months: 2 } );
+```
+
+### _Methods_
+#### `Duration.fromMillis( milliseconds, locale = DEFAULT_LOCALE )`
+
+Creates an instance of `Duration` from a number of milliseconds.
+
+#### `Duration.fromObject( values, locale = DEFAULT_LOCALE )`
+
+Creates an instance of `Duration` from a simple object.  Functionally, this has identical behaviour to `new Duration( values, locale )`
+
+#### `Duration.fromISO( ISOString, locale = DEFAULT_LOCALE )`
+
+Creates an instance of Duration from an [`ISO8601` duration string](https://en.wikipedia.org/wiki/ISO_8601#Durations).
```

Comment:
backticks plus link looks weird in the browser and doesn't appear to be necessary, 

how about:

> Creates an instance of Duration from an [ISO 8601 duration string]